README.MD

```diff
@@ -1 +1,127 @@
+}
+```
+---
+### Python Tutorial
+```json
+{
+  "Topic":"A delve into more complex Python topics, such as loops and functions",
+  "Dates":"8/10/24",
+  "Lecturers":"Cheryl and Kaavya"
+}
+```
+---
+### Data Storytelling + Intro to EDA 
+```json
+{
+  "Topic":"Understanding how data creates narratives and learning to analyse data to gain insights with Python",
+  "Dates":"10/10/24",
+  "Lecturers":"Jun Yeow and Geniveve"
+}
+```
+---
+### Intro to Tableu
+```json
+{
+  "Topic":"A guide to using the data visualsation software Tableu for beginners",
+  "Dates":"15/10/24",
+  "Lecturers":"Jun Yeow and Geneiveve"
+}
+```
+---
+### EDA + Tableu tutorial
+```json
+{
+  "Topic":"Combining Exploraty Data Analysis techniques with Tableu software to create advanced data visualisations",
+  "Dates":"17/10/24",
+  "Lecturers":"Jun Yeow and Geneiveve"
+}
+```
+---
+### Intro to SQL, Intermediate SQL, SQL tutorial
+```json
+{
+  "Topic":"A beginner to advanced SQL course across three lecturess",
+  "Dates":"22/10/24", "24/10/24", "29/10/24"
+  "Lecturers":"Jun Yeow and Geneiveve"
+}
+```
+---
+### Amazon Web Services Workshop
+```json
+{
+  "Topic":"TBD",
+  "Dates":"5/11/24", "7/11/24"
+  "Lecturers":"Jun Yeow, Luqman, Jaryl, Abhishek, Sibikrishknan, Geneiveve, Kaavya, and Cheryl"
+}
+```
+---
+### Intro to Machine Learning, Machine Learning Tutorial
+```json
+{
+  "Topic":"A beginner to advanced course on utilizing Machine Learning techniques across four lectures",
+  "Dates":"12/11/24", "14/11/24", "19/11/24","21/11/24", "26/11/24"
+  "Lecturers":"Luqman and Sibikrishknan"
+}
+```
+---
+### Web scraping
+```json
+{
+  "Topic":"A crash course on web scraping techniques using Python",
+  "Dates":"3/12/24",
+  "Lecturers":"Abhishek and Jaryl"
+}
+```
+---
+### Intro to NLP, NLP tutorial 
+```json
+{
+  "Topic":"A beginner to advanced course on understanding and using Natural Language Processing techniques",
+  "Dates":"5/12/24", "10/12/24", 
+  "Lecturers":"Abhishek and Jaryl"
+}
+```
+---
+### Internal Projects Session
+```json
+{
+  "Topic":"Subcomms will be divided into teams and tasked to create a project which solves a certain problem or fulfills a certain requirment within the deadline. The entire group will then be asked to present their submission to the rest of the club.",
+  "Dates":"27/11/24 to 18/12/24"
+}
+```
+---
+
+## Setting up VSCode
+1. Install Visual Studio Code from https://code.visualstudio.com/
+
+2. Install Anaconda from https://www.anaconda.com/download
+
+3. Open Visual Stuido Code, go to the extensions tab, and download the "Python" extension.
+
+4. After you're done downloading "Python", download the "Jupyter" extension.
+
+5. Congratulations! You are now ready to code in Python as well as able to use Jupyter notebooks.
+
+6. If you need to download data files for your curicullumn lecture, you can find it here: https://drive.google.com/drive/folders/1PIjOqFCnovo8kGbGQDfjFE17BAl_PtqF?usp=drive_link
+
+## Forking the curicullumn
```

Comment:
Can you send the scribe link here for them to view for the forking of the repository, thank you! I think the scribe will provide more clarity